frontends/mit-learn/src/pages/HomePage/TestimonialsSection.tsx

```diff
@@ -274,14 +279,14 @@ const SlickCarousel = () => {
           variant="inverted"
           onClick={slick?.slickPrev}
         >
-          <RiArrowLeftLineStyled />
+          <RiArrowLeftLineStyled aria-hidden />
```

Comment:
Voiceover, at least, (by default) reads `<button aria-label="Some Label"> <svg ></svg> </button>` as "Button group, Some Label". In order to be read as a button, not "button group" the svg needs aria-hidden.

*This makes me understand better way MUI's icons all come with aria-hidden by default.*